manila/share/manager.py

```diff
@@ -135,6 +135,14 @@
                 default=False,
                 help='Offload pending share ensure during '
                      'share service startup'),
+    cfg.IntOpt('force_delete_time',
+               default=2,
+               min=1,
+               max=12,
+               help='Maximum time diff in hours to decide whether share will '
```

Comment:
Can you please add that in case "force" deletion fails -> the regular deletion will be performed, e.g. volume will stay in the recovery queue.

Reply:
updated help section